packages/blocks/src/headers/logo.js

```diff
@@ -96,7 +96,7 @@ applyPropertyControls(HeaderLogo.Logo, linkControls)
 applyPropertyControls(HeaderLogo.Link, linkControls)
 
 HeaderLogo.usage = `
-  <HeaderLogo>
+  <HeaderLogo
```

Comment:
Probably don't want to commit this 😇 everything else looks great! Wonderful job 🎉